From 3bf7cba0677fd2c9e7d05a9945d550abb3d4bd38 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 4 May 2016 18:28:51 +0300
Subject: [PATCH 1124/1240] dts: 8040: enable gop and ppv22 for cp110-slave

- enable gop and ppv22 for cp110-slave in dts
- set KR-10G mode for egiga0 (CP0) & egiga3 (CP1)

Change-Id: Id80bbafb7dc3622777d37f6d30d5e5d35b35fc63
Reviewed-on: http://vgitil04.il.marvell.com:8080/29506
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/dts/armada-80x0-db.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/dts/armada-80x0-db.dtsi b/arch/arm/dts/armada-80x0-db.dtsi
index 034bd09..0bff3ad 100644
--- a/arch/arm/dts/armada-80x0-db.dtsi
+++ b/arch/arm/dts/armada-80x0-db.dtsi
@@ -167,7 +167,7 @@
 					phy-mode = "sgmii"; /* lane-0 */
 					phy = <&phy2>;
 					phy-speed = <2500>;
-					status = "okay";
+					status = "disabled";
 				};
 				mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
@@ -180,7 +180,7 @@
 					status = "okay";
 				};
 				eth1@020000 {
-					status = "okay";
+					status = "disabled";
 				};
 				eth2@030000 {
 					status = "okay";
@@ -291,8 +291,8 @@
 			};
 			gop {
 				mac0 {
-					phy-mode = "sgmii"; /* lane-2 */
-					status = "disabled";
+					phy-mode = "kr"; /* lane-2 */
+					status = "okay";
 				};
 				mac2 {
 					phy-mode = "sgmii"; /* lane-0 */
@@ -303,19 +303,19 @@
 				mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
 					phy = <&phy5>;
-					status = "disabled";
+					status = "okay";
 				};
 			};
 			ppv22@000000 {
-				status = "disabled";
+				status = "okay";
 				eth0@010000 {
-					status = "disabled";
+					status = "okay";
 				};
 				eth1@020000 {
 					status = "disabled";
 				};
 				eth2@030000 {
-					status = "disabled";
+					status = "okay";
 				};
 			};
 		};
-- 
1.9.1

